crates/conjure_rules/src/rule_set.rs

```diff
@@ -0,0 +1,97 @@
+use crate::get_rules;
+use conjure_core::rule::Rule;
+use std::collections::HashMap;
+use std::hash::Hash;
+use std::sync::OnceLock;
+
+/**
+ * A set of rules with a name, priority, and dependencies.
+ *
+ * # Fields
```

Comment:
(Pedantic) Fields of a struct are documented above each field - see https://doc.rust-lang.org/rust-by-example/meta/doc.html